### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: config/application.properties.sample [hunk 1]
```diff
@@ -22,7 +22,7 @@ spring.mail.port=
 
 # THYMELEAF (ThymeleafAutoConfiguration)
 spring.thymeleaf.check-template-location=true
-spring.thymeleaf.prefix=classpath:/templates/
+spring.thymeleaf.prefix=file:./config/templates/
```
Comment (TimvdLippe) [inline config/application.properties.sample]: Already fixed in #318?

Comment (martijnjanssen) [inline config/application.properties.sample]: Yeah, I saw that after a rebase, I am removing this.

File: README.md [hunk 2]
```diff
@@ -1,62 +1,47 @@
...
+  - `spring.datasource.[…]` (`url`, `username`, `password`) to your database url and credentials
+  - `flyway.enabled` to `false`
+  - `a5l.molliekey` and `a5l.googleMapsAPIkey` to their respective keys if you have those
+  - all occurrences of `areafiftylan.nl` to `localhost:5100`
+ 
+  You should comment out:
+  - `spring.jpa.properties.hibernate.hbm2ddl.auto`
+  - `logging.config`
+  
+  You should comment in:
```
Comment (martijnjanssen) [inline README.md]: I would change 'comment in' to 'comment' and 'comment out' to 'uncomment'.

File: README.md [hunk 3]
```diff
@@ -1,62 +1,47 @@
...
+  - `spring.jpa.hibernate.ddl-auto`
+
+4.  Go to the Run/Debug Configuration window `Run -> Edit Configurations`, and add a new Spring Boot configuration. It needs only two settings to run. The `Main class` should be set to `ch.wisv.areafiftylan.Application` and the `Use classpath of module` should be set to your project module. Finally, enable the dev profile by entering `dev` in the `Active Profiles` box.
```
Comment (martijnjanssen) [inline README.md]: You could do this even faster by just right clicking the `Application` file and clicking **Run**. The only thing left to do is filling in the `dev` profile.

File: README.md (hunk 2 above)
Comment (FrancisBehnen) [inline README.md]: Other way around :p, but yea, I get you

**Pull request merged**